internal/flags/helpers.go

```diff
@@ -132,7 +133,7 @@ func (a ByCategory) Less(i, j int) bool {
 func FlagCategory(flag cli.Flag, flagGroups []FlagGroup) string {
 	for _, category := range flagGroups {
 		for _, flg := range category.Flags {
-			if flg.GetName() == flag.GetName() {
+			if reflect.DeepEqual(flg.Names(), flag.Names()) {
```

Comment:
...

Reply:
`GetName()` was removed and now its `Names()` and I'm not sure how should we handle this since it's a slice now.

Reply:
Perhaps `String()` ?